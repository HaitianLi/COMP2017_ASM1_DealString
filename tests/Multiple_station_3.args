Sydney Beijing 11:32:21
