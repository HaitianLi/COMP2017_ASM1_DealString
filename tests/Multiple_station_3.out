The next train to Beijing from Sydney departs at 10:30:30
